misaligned.c

```diff
@@ -16,6 +16,11 @@ int printColorMap() {
 int main() {
     int result = printColorMap();
     assert(result == 25);
+
+    assert(result == 0);

```

Comment:
We need to assert what is expected of the production code. Do we expect 0, 10000 and is it right output?
We must use asserts to explicitly state the behavior of the production code.